Library/Homebrew/os.rb

```diff
@@ -22,5 +22,7 @@ def self.linux?
     require "os/linux"
     ISSUES_URL = "https://github.com/Linuxbrew/brew/wiki/troubleshooting".freeze
     PATH_OPEN = "xdg-open".freeze
+  else
+    ::HOMEBREW_DEFAULT_PREFIX = "/usr/local".freeze
```

Comment:
Feels a bit weird for this to be defined here but not for linux/mac? Does something like `HOMEBREW_DEFAULT_PREFIX ||= "/usr/local".freeze` work in this file and then it could perhaps be omitted on Linux? I dunno the best thing really just this feels odd.

Reply:
Yeah, it felt a bit odd to me to. My best idea was to create an `os/generic` file. I like your idea though.